public/scss/pages/PlaceorderV2.module.scss

```
@@ -0,0 +1,487 @@
+@import '../utils/_variables.scss';
+@import '../components/Buttons.module.scss';
+
+.addressMapButtonMapContainer
+{
+  @include flex(column, flex-start, flex-start);
+  position: relative;
+  margin: 0 0 16px;
+  padding: 10px 12px;
+  border: none;
+  border-radius: 3px;
+  overflow: hidden;
+  background-color: $color_lighter_blue;
+  background-repeat: no-repeat;
+  background-position: top right;
+  background-image: url("data:image/svg+xml,%3Csvg width='42' height='42' viewBox='0 0 35 36' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cg opacity='0.5'%3E%3Cpath d='M35.9628 23.0047L23.5 35.4675L11.0372 23.0047C8.5723 20.5398 6.8937 17.3993 6.21365 13.9804C5.5336 10.5615 5.88265 7.01768 7.21665 3.79715C8.55065 0.576612 10.8097 -2.17602 13.7081 -4.11267C16.6065 -6.04932 20.0141 -7.08301 23.5 -7.08301C26.9859 -7.08301 30.3935 -6.04932 33.2919 -4.11267C36.1903 -2.17602 38.4494 0.576612 39.7834 3.79715C41.1174 7.01768 41.4664 10.5615 40.7864 13.9804C40.1063 17.3993 38.4277 20.5398 35.9628 23.0047V23.0047ZM23.5 14.4585C24.5388 14.4585 25.535 14.0459 26.2695 13.3113C27.004 12.5768 27.4167 11.5806 27.4167 10.5418C27.4167 9.50308 27.004 8.50686 26.2695 7.77234C25.535 7.03782 24.5388 6.62517 23.5 6.62517C22.4612 6.62517 21.465 7.03782 20.7305 7.77234C19.996 8.50686 19.5833 9.50308 19.5833 10.5418C19.5833 11.5806 19.996 12.5768 20.7305 13.3113C21.465 14.0459 22.4612 14.4585 23.5 14.4585Z' fill='%2381CDE9'/%3E%3C/g%3E%3C/svg%3E%0A");
+}
+
+.addressMapButtonMapLabel
+{
+  @include flex(row, center, flex-start);
+  display: inline;
+  position: relative;
+  margin: 0 4px 0 0;
+  padding: 0;
+  white-space: normal;
+  word-break: break-all;
+  font-size: 12px;
```

```suggestion
  @include fontSize(12);
```

pakay mixin ini pacc